### DISCUSSION TITLE
kaleidoscope/

### DISCUSSION THREAD

**Discussion by giscus** [zidekuls/kaleidoscope#2] [Announcements]
# kaleidoscope/

https://zidekuls.github.io/kaleidoscope/

**Comment by zidekuls** [comment]
![](https://hamibot.com/illustrations/home.png)

**Comment by zidekuls** [comment]
天木蓼 @ 2022-09-11 16:49:055320014

刚生娃的女人告诉我婚姻80%的事都要谈钱，40多岁的女人告诉我结婚要找有钱的。我又看过太多没钱的，未婚先孕的，房子又远又小，日子过的惨兮兮，只是强装幸福。所以找对象什么重要不言而喻。

[举报]OO [65] XX [14] 吐槽 [7]

热评

晚秋的霜 广东省深圳市

我建议男性找门当户对的

#10979265@1天前OO[82]XX[0]举报

想要香香 江苏省苏州市

是男是女都想要找有钱的，可有那资本找吗？而且能找的到有钱又心性好的吗？至少我公司一个20岁的妹子，身材棒，颜值是那种网红开了滤镜才有的又白又嫩，那时候是个男的都想凑过去。结果呢？嫁给一个家里做（比批发大，比贸易小，突然卡壳了，万能的蛋友们，有什么词能形容的？）这种生意的，妹子又做饭又做家务，他们找不到缺点居然说她赚的少。后来她忍不了就离婚了。还有我经理，就是上门女婿，结婚之后3天一小吵，5天一大吵，只是不离婚，为了什么？他跟我说了，开始不吵，你就没地位。结果呢，吵了快10年了，2个孩子都上小学了，前阵子吵架的时候推了他老婆一下，他老婆直接报警说他打她。真就以为软饭好吃吗？

#10979277@1天前OO[59]XX[0]举报

HL 江苏省苏州市

作为一个已婚男人，我也这么觉得。如果自己先天条件优秀一点，找个更有钱的老婆，那我就不用苦哈哈加班，定时上交工资。想买的驾驶模拟器、按摩椅，肯定也存够工资买下来了。

#10979282@1天前OO[20]XX[1]举报

狼狼 重庆市

金钱是物资或时间的支配权

#10979293@1天前OO[11]XX[0]举报

wayne 广东省深圳市

为什么很多人都会把问题简单化？婚姻的基础难道就是简简单单一样东西就搞定的吗？
建房子要打地基，你说地基什么最重要，是水泥、沙子、钢筋、还是什么别的？你只要一样行不行？

#10979389@1天前OO[10]XX[1]举报

**Comment by zidekuls** [comment]
白菜 @ 2022-09-12 02:44:585320255
说起家里蹲这个词，想起四五年前在老家有个找我家租了个单间的25岁左右的男的，一个月150房租电费1块一度，水免费，不过宽带是他自己装的，一个月100多好像。他每天都把自己关起来，除了上厕所洗衣服便不出门，每天开很多个号玩魔域还是魔什么的游戏赚钱，他跟我说只要不被封号封脚本什么的，一般一个月2-3K有的，封了就不好说了。但是逢年过节的时候他也会去买菜，请1-2个老乡在房间里用电饭煲炖排骨喝啤酒啥的，自给自足外还能保持一定的社交。以前感觉他很惨，现在自己当了6-7年社畜反而有点羡慕他，尤其是发现他是真的爱玩那个游戏。

[举报]OO [84] XX [1] 吐槽 [2]
热评
黄油饺子 贵州省贵阳市
我觉得最让人羡慕的不是他可以做自己喜欢的事情赚钱，而是他知道自己想要什么，可以不在意别人看法的活着
#10979763
@12小时前OO[33]XX[1]举报

**Comment by zidekuls** [comment]
```javascript
const { APP_ENV, msg } = hamibot.env;
log('消息:', msg);
```

**Comment by zidekuls** [comment]
![IMG_20220917_172946_424](https://user-images.githubusercontent.com/28500463/190863600-cf2b5cce-0bdc-439e-8b0e-9895f69b65c0.jpg)

**Comment by zidekuls** [comment]
![](https://gitlab.com/zidekuls/tempmock/uploads/65565b15a99340ddd37b336c4c643caf/416490211.jpg)

**Comment by zidekuls** [comment]
暧昧
有一部分人只适合相亲，不适合自由恋爱，如果自由她就一直找到错的人，就和炒股一样，不深套不死心
#10505330
@1天前OO[11]XX[0]举报

**Comment by zidekuls** [comment]
| A   | B   | C   |
| --- | --- | --- |
| A1  | B1  | C1  |
| A2  | B2  | C2  |

- [x] 起床
* [ ] 洗漱
+ [ ] 吃早餐

::: info 蓝蓝的 :::

::: spoiler 展开
折叠的内容
:::

**Reply by zidekuls** [reply]
◑.◑

**Reply by zidekuls** [reply]
(꒪ȏ꒪;)

**Reply by zidekuls** [reply]
｡°(°¯᷄◠¯᷅°)°｡

**Comment by zidekuls** [comment]
S081335 @ 2022-05-01 13:18:365223188
我更喜欢青少年时，对一切都感兴趣，永远充满好奇心的时候
到二十多岁后，这种好奇心每日都在减少，我就像被包裹了无数层塑料膜一样接触着这个世界
也许是因为自己有了标准，有了审美与观点，能达到自己预期的事物越来越少，但好在我还有好奇心，我见过比我更年轻的，却早早对一切事物失去兴趣，不知道自己会喜欢什么，也不去建立标准的人。
我每天都在避免自己成为这样的人。

[举报]OO [14] XX [3] 吐槽 [3]
#1楼ฅ՞•ﻌ•՞ฅ
今天同学聚会带了几个2，3岁的小孩子，他们滑滑梯可以玩半个小时，看见蝴蝶飞走飞来会开心失望。人越是学习知识越是会失去这种好奇心，最后都变成普通人。
#10412191
@3小时前OO[2]XX[1]举报
#2楼晚睡早起土法死掉
我喜欢年轻时视力好，能张目对日明察秋毫，嗅觉也灵敏，下过雨总能闻到泥土的芬芳
#10412205
@3小时前OO[4]XX[0]举报
#3楼Anmu
希望你还房贷之后还能这样
#10412206
@3小时前OO[0]XX[0]举报

**Comment by zidekuls** [comment]
https://ws.l3on.site/post/20/

**Comment by zidekuls** [comment]
@octocat :+1: This PR looks great - it's ready to merge! :shipit:
